avalanched-aws/src/agent/mod.rs

```diff
@@ -70,6 +72,14 @@ pub fn command() -> Command {
                 .required(false)
                 .num_args(0),
         )
+        .arg(
```

Comment:
Like we discussed, we don't want to use profile, we use https://github.com/ava-labs/avalanche-ops/blob/main/avalanche-ops/src/aws/cfn-templates/ec2_instance_role.yaml.